src/data/resolver/hello-world.resolver.ts

```diff
@@ -0,0 +1,10 @@
+import { Resolver, Query } from 'type-graphql';
+import { UserEntity } from '../entity/user.entity';
+
+@Resolver((of) => UserEntity)
```

Comment:
Esse parâmetro do `@Resolver` não é obrigatório e geralmente não usamos no projeto. Sabe se existe alguma vantagem em usar ele? Confesso que nunca pesquisei sobre 🤔 

Aliás, esse resolver não é de usuário, será que ele deveria ter esse parâmetro definido mesmo?

Reply:
Esses types no Resolver estão aí por motivos de Ctrl+C, Ctrl+V.
De fato isso é completamente inútil nesse caso. Pelo que entendi dá pra declarar esse tipo no decorator pra usar o @FieldResolver, que é um decorator usado para criar resolvers para os campos que são relações.